.editorconfig

```diff
@@ -0,0 +1,28 @@
+# http://editorconfig.org
+
+root = true
+
+[*]
+charset = utf-8
+end_of_line = lf
+insert_final_newline = true
+trim_trailing_whitespace = true
+
+[*.go]
+indent_style = tab
+indent_size = 8
+
+[*.tmpl]
```

Comment:
I don't really think they are cumbersome. In fact it makes it more easy to just copy paste `.editorconfig` from gitea into all other projects and create PRs.